### DISCUSSION TITLE
Incompatible provider version jfrog/artifactory

### DISCUSSION THREAD

**Discussion by vasanthig21** [jfrog/terraform-provider-artifactory#239] [Q&A] [answered]
It is not possible to automatically install the provider via terraform init on linux_amd64

<img width="769" alt="Screen Shot 2021-12-10 at 7 16 01 PM" src="https://user-images.githubusercontent.com/94625595/145656389-a942f6bf-80ca-4f5a-8806-76ba80cf6fc9.png">

As a workaround, I tried with all versions of jfrog/artifactory.  But all of them are giving same error and also tried with commenting the version in jfrog/artifactory provider block. Still the same error. 

Can anyone help me with this issue?

below is the versions.tf file where versions of required providers were declared.

```tf
terraform {
  required_version = "~> 1.0"
  backend "remote" {
    hostname     = "app.terraform.io"
    organization = "platformzero"
    workspaces {
      name = "tf-artifactory-cloud"
    }
  }

  required_providers {
    artifactory = {
      source  = "jfrog/artifactory"
      #version = "2.6.21"
    }
  }
}

provider "azurerm" {
  #version         = "= 2.89.0"
  subscription_id = var.subscription_id
  tenant_id       = var.tenant_id
  features {}
}
```

**Comment by alexhung** [comment, marked as answer]
@vasanthig21 Sorry I missed this until now. Is this still an issue for you? 2.2.5 is a older version released back in Dec 2020. Will you be able to upgrade to a later version that has `linux_amd64`?

**Comment by alexhung** [comment]
@vasanthig21 I'm closing this discussion due to lack of response. If you are still unable to install the provider, please open an issue.
